src/main/java/org/myrobotlab/service/DiyServo.java

```diff
@@ -360,7 +360,7 @@ public long getLastActivityTime() {
     return lastActivityTimeTs;
   }
 
-  public double getPos() {
+  public double getcurrentOutputPos() {

```

Comment:
this method goes.. nothing calls it.. it's not named properly. it should inherit.